07.bitstream.semantics.md

```diff
@@ -461,9 +461,11 @@ that subsampling_x is equal to 0 and subsampling_y is equal to 0.
 | 0                      | CSP_UNKNOWN                   | Unknown (in this case the source video transfer function must be signaled outside the AV1 bitstream)
 | 1                      | CSP_VERTICAL                  | Horizontally co-located with (0, 0) luma sample, vertical position in the middle between two luma samples
 | 2                      | CSP_COLOCATED                 | co-located with (0, 0) luma sample
-| 3                      | CSP_RESERVED                  |
+| 3                      | CSP_CENTER                    | In the middle between four luma samples
 {:.table .table-sm .table-bordered }
 
+**Note:** This field may be overridden by future metadata specifications.
```

Comment:
The original reason for this note was because we ended up using up all of the bits available with this addition. But I agree, it's incredibly unlikely we would add any other chroma positions.